controllers/component_build_controller_simple_build.go

```diff
@@ -83,12 +88,32 @@ func (r *ComponentBuildReconciler) SubmitNewBuild(ctx context.Context, component
 	simpleBuildPipelineCreationTimeMetric.Observe(time.Since(component.CreationTimestamp.Time).Seconds())
 
 	log.Info(fmt.Sprintf("Build pipeline %s created for component %s in %s namespace using %s pipeline from %s bundle",
-		buildPipelineRun.Name, component.Name, component.Namespace, pipelineRef.Name, pipelineRef.Bundle),
+		buildPipelineRun.Name, component.Name, component.Namespace, pipelineName, pipelineBundle),
 		l.Action, l.ActionAdd, l.Audit, "true")
 
 	return nil
 }
 
+func getNameAndBundle(pipelineRef *tektonapi.PipelineRef) (string, string, error) {
```

Comment:
can we have it in component_build_controller_common.go ?

Reply:
makes sense, moved